### PR TITLE
Reader: Implement manage settings pop-out component

### PR DESCRIPTION
**Pull request by samouri**
Implement the settings dialog as show here:

Implementation notes:
- This is just the UI for the component.  `connect`ing it to actual data / reducers will happen in a future pr.
- couldn't figure out how to make background of the arrow in the popover become the right color without resorting to crazy hacky css.

### CODE REVIEW

File: client/blocks/reader-subscription-list-item/email-settings.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+				<Popover
+					onClose={ this.props.onClose }
+					isVisible={ this.state.showPopover }
+					context={ this.spanRef }
+					position={ 'bottom left' }
+				>
+					<div className="reader-subscription-list-item__email-popout-wrapper">
+						<h3 className="reader-subscription-list-item__email-popout-header"> { translate( 'Email me' ) } </h3>
+						<div>
+							<span> { translate( 'New posts' ) } </span>
```
Comment (a8ci18n) [inline client/blocks/reader-subscription-list-item/email-settings.jsx]: Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'new posts', { context: 'text for multiple new posts notification (2 new posts)'} )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/settings-popout)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/reader-subscription-list-item/email-settings.jsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+					position={ 'bottom left' }
+					className="reader-subscription-list-item__settings-menu-popout"
+				>
+					<div className="reader-subscription-list-item__email-popout-wrapper">
+						<h3 className="reader-subscription-list-item__email-popout-header">
+							{ translate( 'Email me' ) }
+						</h3>
+						<hr className="reader-subscription-list-item__header-separator" />
+						<div className="reader-subscription-list-item__email-popout-toggle">
+							{ translate( 'New posts' ) }
```
Comment (a8ci18n) [inline client/blocks/reader-subscription-list-item/email-settings.jsx]: Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'new posts', { context: 'text for multiple new posts notification (2 new posts)'} )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/settings-popout)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by bluefuton** [comment]
Any testing instructions for this one @samouri? Should this work from http://calypso.localhost:3000/devdocs/blocks/reader-subscription-list-item?

**Comment by samouri** [comment]
Yes @bluefuton!
Going to https://calypso.live/devdocs/blocks/reader-subscription-list-item?branch=add/reader/settings-popout and clicking the cog should make it appear.   Because the toggles are controlled from redux state they shouldn't be usable yet and finalizing the styles will also happen in a separate PR.

**Comment by bluefuton** [comment]
> For now just appearing, disppearing, and resembling the mocks is good enough

🚢🚢🚢

**Pull request merged**